src/components/Form/Form.tsx

```diff
@@ -0,0 +1,161 @@
+/* eslint-disable */
+import React, { FC, useState } from "react";
+import { Formik, Field } from "formik";
+import {
+  Button,
+  LinearProgress,
+  MenuItem,
+  Snackbar,
+  FormControl,
+  InputLabel,
+} from "@material-ui/core";
+import { TextField, Select } from "formik-material-ui";
+import * as yup from "yup";
+import FormContainer from "components/FormContainer/FormContainer";
+import {
+  createStyles,
+  makeStyles,
+  useTheme,
+  Theme,
+} from "@material-ui/core/styles";
+
+export interface FormData {
+  email?: string;
+  password?: string;
+  select?: string;
+  //etc...
+}
+
+const FormSchema = yup.object().shape({
+  email: yup.string().email("Invalid email").required("Required"),
+  password: yup.string().min(8, "Password too short!").required("Required"),
+  //etc...
+});
```

Comment:
remove this, this should be added when the `Form` component is being consumed by other components